lib/util/probe.js

```diff
@@ -32,30 +32,64 @@ function startProbeServer(config, callback) {
     probeServer.start();
 }
 
+/**
+ * Send an Error response with multiple errors
+ * @param {http.HTTPServerResponse} res - HTTP response for writing
+ * @param {Logger} log - Werelogs instance for logging if you choose to
+ * @param {Object} errorMessages - error messages
+ * @return {undefined}
+ */
+function sendMultipleErrors(res, log, errorMessages) {
+    const messages = JSON.stringify(errorMessages);
+    const errorCode = 500;
+    log.error('sending back error response', {
+        httpCode: errorCode,
+        error: messages,
```

Comment:
It's not ideal to log JSON-serialized contents, as it will be serialized again so not very readable as is.

It's likely that for indexing purpose we want to keep the `error` field as a string.

Maybe we can separate this logic in two cases:
- if there is a single error, just log its message in `error`
- if there is more than one error, add `errors: errorMessages` to the log, and have `error` log something generic like `multiple errors (see "errors" field for details)`. Alternatively we may also concatenate error messages in a "human-readable" string but we may end up with a long error message, not necessarily very readable either.

That would be my recommendation but I leave it up to you.